components/ws-manager/pkg/manager/manager.go

```diff
@@ -117,23 +117,16 @@ const (
 	stopWorkspaceNormallyGracePeriod = 30 * time.Second
 	// stopWorkspaceImmediatelyGracePeriod is the grace period we use when stopping a pod as soon as possbile
 	stopWorkspaceImmediatelyGracePeriod = 1 * time.Second
-	// wsdaemonDialTimeout is the time we allow for trying to connect to ws-daemon.
-	// Note: this is NOT the time we allow for RPC calls to wsdaemon, but just for establishing the connection.
-	wsdaemonDialTimeout = 10 * time.Second
```

Comment:
Any reason why we remove this?
We still use 10 seconds timeout when dialing to ws-daemon in function `newWssyncConnectionFactory`.